js/components/slider.js

```diff
@@ -0,0 +1,61 @@
+var React = require('react'),
+ReactDOM = require('react-dom'),
+Utils = require('./utils');
+
+var Slider = React.createClass({
+
+  componentDidMount: function() {
+    this.handleSliderColoring();
+  },
+
+  componentWillReceiveProps: function(nextProps) {
+    if (nextProps.value != this.props.value) {
+      this.handleSliderColoring();
+    }
+  },
+
+  handleSliderColoring: function(){
+    if (!Utils.isEdge()){
+      var inputs = document.querySelectorAll('input[type=range]');
```

Comment:
You fixed it before I could finish reviewing the whole thing, is there a reason we can't use refs instead of a DOM query? What's the case where we expect multiple sliders?